tests/functional/plugincheck.js

```diff
@@ -81,3 +81,19 @@ casper.test.begin('PluginCheck, outated Firefox: ' + url, 1, function suite(test
         helpers.done();
     });
 });
+
+casper.test.begin('PluginCheck, no plugins found message: ' + url, 2, function suite(test) {
+    casper.start();
+
+    casper.userAgent(config.userAgent.fx);
+
+    casper.thenOpen(url, function() {
+        test.assertVisible('#no-plugins', 'The no plugins container is visible');
```

Comment:
Yes, that is the assumption so, will add a comment about that. Whether this will always be the case, well, I believe so. Casper does have a flag to tell it to load plugins but, having tried that previously it did not seem to work for me. With that said then, seeing that, that flag is false by default, this would be a safe assumption:

http://docs.casperjs.org/en/latest/modules/casper.html#pagesettings